### DISCUSSION TITLE
Skip parts that are cut out in between segments?

### DISCUSSION THREAD

**Discussion by rRobis** [mifi/lossless-cut#1581] [Q&A] [answered]
Hello!

Is it possible to somehow playback the timeline, but skip the parts that have trash bin on them? I tried to find some option, but couldn't find.

I need to cut out some parts from video to make it shorter, but I need to preview on the spot how the cut between parts will look like.

Currently when you play the video, it just plays everything, without skipping the trashed parts.

This app is the best in it's field, i'm very thankful for having it!🙂

**Comment by mifi** [comment, marked as answer]
Hey, try the colorful play button in the bottom right

<img width="62" alt="Screenshot 2023-04-23 at 10 23 56" src="https://user-images.githubusercontent.com/402547/233828574-583251b1-e523-4c66-92b1-07aca1bc5edf.png">

**Reply by rRobis** [reply]
Oh, nice, thanks! Somehow I didn't notice it 😅

**Comment by rRobis** [comment]
@mifi Hi! I tried out the function, but there is a small flaw.

If I have a big segment (let's say 30 seconds) and I make a cut in the middle (15-16 seconds). If I press that button, I have to watch the full segment from beginning to get to the next one.

It would be great, if it was possible to toggle this state, and the play/pause button would start where the playhead is, and skip the empty parts.

**Reply by mifi** [reply]
you could try to bind the additional hotkey for this:

- "Loop beginning and end of current segment"

**Reply by rRobis** [reply]
For me this shortcut just starts playing the current segment from the beginning.

Maybe I didn't explain clear enough, but I would need it to work this way:

I have: (The || is playhead)

<-------SEG1----||---> <----- EMPTY space -----> <-------SEG2-------> 

Currently, when pressing rainbow button, or "Loop beginning and end of current segment", it goes to the beginning of SEG1, plays whole SEG1 and jumps over EMPTY space and goes to SEG2 (as expected).

But I need a shortcut (or option to make play/pause work this way), that would start playing where the "||" (playhead) is (not particularly from end, but wherever I put it and start from that position), and jump to the SEG2 (like the rainbow button does). 

Most importantly, that it wouldn't start from the beginning of current segment, but start from wherever the playhead currently is.

Thanks for your replies! 🙂

**Reply by mifi** [reply]
created a feature request for this #1584

**Reply by rRobis** [reply]
thanks!
